test/integration/registration/integration_suite_test.go

```diff
@@ -37,8 +37,12 @@ import (
 )
 
 const (
-	eventuallyTimeout  = 30 // seconds
-	eventuallyInterval = 1  // seconds
+	eventuallyTimeout     = 30 // seconds
+	eventuallyInterval    = 1  // seconds
+	testCustomLabel       = "custom-label"
```

Comment:
this should not be set as global variable. It will be really hard to maintain. You should consider stop the existing hub, customize the option and restart the hub in the specific test. Example like this https://github.com/open-cluster-management-io/ocm/blob/main/test/integration/registration/spokecluster_aws_joining_test.go#L32-L47